v0/destinations/zendesk/data/ZDIdentifyConfig.json

```diff
@@ -10,8 +10,8 @@
   {
     "destKey": "user.name",
     "sourceKeys": [
-      "traits.name",
-      "context.traits.name"
+      "context.traits.name",
```

Comment:
This should be handled separately for group calls. this will create wrong output for identify calls if precedence is changed.